extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -535,10 +548,11 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
     let psz = null;
     let fws = null;
     if (this.sendFlexibleAdSlotParams_) {
-      const {fwSignal, slotWidth, parentWidth} = getFlexibleAdSlotRequestParams(
+      this.flexibleAdSlotData_ = getFlexibleAdSlotData(
         this.win,
         this.element.parentElement
       );
+      const {fwSignal, slotWidth, parentWidth} = this.flexibleAdSlotData_;
```

Comment:
I honestly don't know the answer to this, but...what happens when you try to destructure null?

Reply:
Probably a runtime error. But in this case, the result is guaranteed to be defined.